### ISSUE TITLE
[BUG] Can't assign ID to resource, package already has ID

### ISSUE THREAD

**Issue by catweaselirl** [iBotPeaches/Apktool#3218]
### Information
1. **Apktool Version (`apktool -version`)** - 2.8.1
2. **Operating System (Mac, Linux, Windows)** - Windows
3. **APK From? (Playstore, ROM, Other)** - APKmirror
4. **Java Version (`java --version`)** - 20.0.1

### Stacktrace/Logcat

```
java -jar C:\Users\lower\.apklab\apktool_2.8.1.jar b c:\Users\lower\AppData\Local\apk-mitm\Cache\walmart --use-aapt2
I: Using Apktool 2.8.1
I: Checking whether sources has changed...
I: Checking whether sources has changed...
I: Checking whether sources has changed...
I: Checking whether sources has changed...
I: Checking whether sources has changed...
I: Checking whether sources has changed...
I: Checking whether sources has changed...
I: Checking whether sources has changed...
I: Checking whether sources has changed...
I: Checking whether resources has changed...
I: Building resources...
W: error: can't assign ID 0x80010000 to resource com.walmart.squiggly.compass_dynamic_module:anim/nav_default_enter_anim because package already has ID 80.
W: error: failed assigning IDs.
brut.androlib.exceptions.AndrolibException: brut.common.BrutException: could not exec (exit code = 1)

```

### Steps to Reproduce
1. Decompile the latest "Me@Walmart" app
2. Build with aapt2

### Frameworks
If this APK is from an OEM ROM (Samsung, HTC, LG). Please attach framework files
(`.apks` that live in `/system/framework` or `/system/priv-app`)

### APK
If this APK can be freely shared, please upload/attach a link to it. https://m.apkpure.com/me-walmart/com.walmart.squiggly/download

### Questions to ask before submission
1. Have you tried `apktool d`, `apktool b` without changing anything? Yes
3. If you are trying to install a modified apk, did you resign it? Couldn't build, so no?
4. Are you using the latest apktool version? Yes

**Comment by iBotPeaches** [comment]
Thanks for the report. This has a fused/dynamic manifest with a new package inside (`com.walmart.squiggly.compass_dynamic_module`).

Since you got to the build step, I'm pretty surprised since Apktool's handling of this situation even on disassembly is quite rough. You'd have to trick it by installing itself as a framework, but then it would probably rename your manifest and more.

tldr - yes this is a known bug/flaw in Apktool and being tracked at - https://github.com/iBotPeaches/Apktool/issues/2960.

Closing for duplicate and should have time to look at it in the new few weeks.

**Comment by catweaselirl** [comment]
> Since you got to the build step, I'm pretty surprised since Apktool's handling of this situation even on disassembly is quite rough. You'd have to trick it by installing itself as a framework, but then it would probably rename your manifest and more.
Would the app build and be functional with this workaround? If so I would really appreciate if you could explain in a little more detail how to do that

**Comment by iBotPeaches** [comment]
It wouldn't. The trick I was referring to was just installing itself as a framework, since it would detect the shared object.

Something like
```
apktool if apk.name -t 3218
apktool d apk.name -t 3218
```
